appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -0,0 +1,120 @@
+$location=$args[0]
```

Comment:
I suggest the name `$logFilePath` since "location" means something else in SARIF. (In any case, variable names with "path" are a common convention when the variable contains a file system path.)